x/fbridge/client/cli/tx.go

```diff
@@ -4,10 +4,14 @@ import (
 	"github.com/spf13/cobra"
 
 	"github.com/Finschia/finschia-sdk/client"
+	"github.com/Finschia/finschia-sdk/client/flags"
+	"github.com/Finschia/finschia-sdk/client/tx"
+	sdk "github.com/Finschia/finschia-sdk/types"
+	sdkerrors "github.com/Finschia/finschia-sdk/types/errors"
 	"github.com/Finschia/finschia-sdk/x/fbridge/types"
 )
 
-// GetTxCmd returns the transaction commands for this module
+// NewTxCmd returns the transaction commands for fbridge module
```

Comment:
It seems like the fbridge namespace isn't visible. Do you need to add it to `AppModuleBasic` for it to be accessible? (my command was: `simd tx fbridge ...` (`fswap` was visible)

Reply:
it's already added. Please recheck